desktop/src/main/java/bisq/desktop/main/portfolio/closedtrades/ClosedTradesView.java

```diff
@@ -263,18 +273,7 @@ public void initialize() {
                     TableRow<Tradable> row = new TableRow<>();
                     ContextMenu rowMenu = new ContextMenu();
                     MenuItem editItem = new MenuItem(Res.get("portfolio.context.offerLikeThis"));
```

Comment:
editItem -> duplicateItem

Reply:
Not from my side, but makes more sense with this naming.